Cabal/src/Distribution/Simple/Glob.hs

```diff
@@ -336,3 +337,8 @@ splitConstantPrefix = unfoldr' step
   where
     step (GlobStem seg pat) = Right (seg, pat)
     step (GlobFinal pat) = Left pat
+
+
+isRecursiveInRoot :: Glob  ->Bool
```

Comment:
␣ slip, should be `Glob -> Bool`.

Reply:
@dyniec: ping? After months of inactivity we are suddenly in a hurry to merge this (release imminent). :)

Reply:
Sorry about delay. I can take a look tonight